tests/mocks/mock_client.py

```diff
@@ -39,17 +40,23 @@ def __init__(self):
         self.patcher_grpc_client = patch(
             'model_analyzer.triton.client.grpc_client.grpcclient.InferenceServerClient',
             Mock(return_value=mock_grpc_client))
+        super().__init__()
+
+    def start(self):
```

Comment:
You don't need the start method anymore. It will automatically iterate on the list.

Reply:
I am overriding the start method to store some variables used in other functions

Reply:
I see. Feel free to ignore my comments.